Asteroid.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Drawing;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DZ1
+{
+    class Asteroid: BaseObject
+    {
+        Image fileOfAsteroid = Image.FromFile("asteroid.png");
```

Comment:
Может быть имеет смысл сделать это поле статическим? Все астероиды будут одинаковыми. Зачем тогда для каждого загружать свою картинку?

```suggestion
        private static readonly Image fileOfAsteroid = Image.FromFile("asteroid.png");
```